tools/testbench/include/testbench/common_test.h

```diff
@@ -118,6 +116,8 @@ void sys_comp_tdfb_init(void);
 void sys_comp_module_demux_interface_init(void);
 void sys_comp_module_eq_fir_interface_init(void);
 void sys_comp_module_eq_iir_interface_init(void);
+void sys_comp_module_mfcc_interface_init(void);
+void sys_comp_module_mixer_interface_init(void);
```

Comment:
if it worked until now without these - does it mean, that this isn't compile-tested by the CI or maybe that these prototypes aren't needed and those functions can be static?

Reply:
I wonder if recent testbench patches have forgotten these? But these will definitely be needed to load the components (unless they get deprecate - e.g. mixer in favor of mixin/mixout?) and run them.

Reply:
@lyakh Sorry, I was wrong. These protypes are no more necessary. The component.h is used, need to delete all these in the patch.